ui/src/shared/copy/notifications.ts

```diff
@@ -99,10 +99,10 @@ export const SetupSuccess: Notification = {
   message: 'Initial user details have been successfully set',
 }
 
-export const SetupError: Notification = {
+export const SetupError= (message: string): Notification => ({
   ...defaultErrorNotification,
```

Comment:
yeah as we discussed, I think it would be better to notify some of our copy "could not do x" followed by message from back end.

Reply:
Got it!